3p/recaptcha.js

```diff
@@ -88,11 +94,11 @@ window.initRecaptcha = function() {
   const {sitekey} = dataObject;
   const recaptchaApiUrl = RECAPTCHA_API_URL + sitekey;
 
-  loadScript(window, recaptchaApiUrl, function() {
-    const {grecaptcha} = window;
+  loadScript(win, recaptchaApiUrl, function() {
+    const {grecaptcha} = win;
```

Comment:
is grecaptcha some sort of global?

Reply:
Yes, grecaptcha is a global provided by the reCAPTCHA API.